episodes/chap4_manage_via_ui.md

```diff
@@ -28,7 +28,9 @@ exercises: 2
 
 Vantage6 uses a permission system to control who can do what in the system. The permission system is based on **roles**, which are collections of rules that define the permissions of a user. A user can have multiple roles, and the permissions of the user are defined by the assigned **rules**.
 
-The permission rules define what each entity is allowed to do, based on the operation (view, create, edit, delete, send, receive), the scope (own, organization, collaboration, global), and the resource (e.g. users, orgarnizations). Users can be assigned anywhere between zero and all of these permission rules. For example, having the rules with `create` in the scope `organization` for the resource `user` means that the user can create users for the organization they belong to.
+The permission rules define what each entity is allowed to do, based on the operation (view, create, edit, delete, send, receive), the scope (own, organization, collaboration, global), and the resource (e.g. users, organizations). Users can be assigned anywhere between zero and all of these permission rules. For example, having the rules with `create` in the scope `organization` for the resource `user` means that the user can create users for the organization they belong to.
+
+Note that not every resource has all scopes and/or operations. For example, the `organization` resource has neither the `own` scope nor the `delete` operation.
```

Comment:
Coincedence: we are adding the `delete` operation for `/organization` in v4.7 so this will no longer be correct at time of the workshop

Reply:
So best to replace this with another example, such as: there is no rule to create collaborations with organization scope, as it does not make sense to create a collaboration that only your own organization can participate in